subsys/bluetooth/mesh/Kconfig

```diff
@@ -59,6 +59,13 @@ config BT_MESH_PROVISIONER
 	help
 	  Enable this option to have support for provisioning remote devices.
 
+config BT_MESH_IV_UPDATE_SEQ_LIMIT
+	hex "the sequence limit to start iv update"
+	default 0x7A1200
```

Comment:
Is there any reason for changing the default from the original `0x800000`?

Reply:
I mistook it for decimal and changed it back to the default value 0x8000000

Reply:
I'm confused. Wasn't the original `8000000` in decimal?